account_invoice_tax_required/tests/test_account_move_tax_required.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2016 - Tecnativa - Angel Moya <odoo@tecnativa.com>
+# Copyright 2024 - Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import exceptions
+from odoo.tests import tagged
+
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged("post_install", "-at_install")
+class TestAccountInvoiceTaxRequired(AccountTestInvoicingCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.account_invoice = cls.env["account.move"]
+        cls.account_journal = cls.env["account.journal"]
+        cls.journal = cls.account_journal.create(
+            {"code": "test", "name": "test", "type": "sale"}
+        )
+        cls.partner = cls.env.ref("base.res_partner_3")
+
+        cls.account_account = cls.env["account.account"]
+        cls.account_rec1_id = cls.account_account.create(
+            dict(
+                code="20000",
+                name="customer account",
+                account_type="asset_receivable",
+                reconcile=True,
+            )
+        )
+        cls.product_product = cls.env["product.product"]
+        cls.product = cls.product_product.create(
+            {
+                "name": "Test",
+                "categ_id": cls.env.ref("product.product_category_all").id,
+                "standard_price": 50,
+                "list_price": 100,
+                "type": "service",
+                "uom_id": cls.env.ref("uom.product_uom_unit").id,
+                "uom_po_id": cls.env.ref("uom.product_uom_unit").id,
+                "description": "Test",
+            }
+        )
+
+        invoice_line_data = [
+            (
+                0,
+                0,
+                {
```

Comment:
Could you use the [Command](https://github.com/odoo/odoo/blob/7dcdb774e61e6d520d49dc402c4fad6dcf527ac1/odoo/fields.py#L4227-L4341) class? It’s more readable.

```suggestion
           Command.create(
                {
```